src/main/java/org/alliancegenome/curation_api/services/helpers/validators/GeneDiseaseAnnotationValidator.java

```diff
@@ -21,6 +20,10 @@
     AffectedGenomicModelDAO agmDAO;
     @Inject
     GeneDiseaseAnnotationDAO geneDiseaseAnnotationDAO;
+    @Inject
+    VocabularyTermDAO vocabularyTermDAO;
+    
+    private String GENE_DISEASE_RELATION_VOCABULARY = "Gene disease relations";
```

Comment:
Now I am seeing that they are defined in multiple classes. They definitely need to get pulled out into a class of their own.

Reply:
Yep, done